pom.xml

```diff
@@ -47,8 +47,8 @@ under the License.
 
   <properties>
     <maven.version>3.0.5</maven.version>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Yeah!

Reply:
Well, actually https://issues.apache.org/jira/browse/MNG-6399 should do that, but I needed it here